app/src/main/java/cm/aptoide/pt/networking/image/ImageLoader.java

```diff
@@ -123,13 +122,14 @@ public static void cancel(Context context, View target) {
         placeHolderDrawableId);
   }
 
-  public Target<Drawable> loadWithCircleTransformAndPlaceHolder(String url, ImageView imageView,
+  private Target<Drawable> loadWithCircleTransformAndPlaceHolder(String url, ImageView imageView,
```

Comment:
Should this be private even though it is only used in this class? The ImageLoader goal is to work like a lib, so in my opinion its contract should be public for anyone using it. Otherwise to use it it would require to change the ImageLoader dependency.